File: 프로그래머스/1/12926. 시저 암호/시저 암호.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

string solution(string s, int n) {
    string answer = "";
    
    for(int i=0; i<s.size(); i++) {
        char c = s[i];
        
        if(c >= 'a' && c <= 'z') {
            answer += (char)((c - 'a' + n) % 26 + 'a');
        } else if(c >= 'A' && c <= 'Z') {
            answer += (char)((c - 'A' + n) % 26 + 'A');
        } else {
            answer += c;
        }
    }
    
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        
        for(int i=0; i<s.length(); i++) {
            char c = s.charAt(i);
            
            if('a' <= c && c <= 'z') {
                answer += (char)('a' + (c - 'a' + n) % 26);
            } else if ('A' <= c && c <= 'Z') {
                answer += (char)('A' + (c - 'A' + n) % 26);
            } else {
                answer += " ";
            }
        }
        return answer;
    }
}